app/Main.hs

```diff
@@ -1,27 +1,90 @@
-{-# LANGUAGE ViewPatterns #-}
-
 module Main where
 
-import           Data.Text              (Text)
-import qualified Data.Text.IO           as TIO
-import           Formatting             (int, sformat, (%))
-import           System.Environment     (getArgs)
+import           Control.Applicative    ((<**>))
```

Comment:
It's better to use latest `.stylish-haskell` from `serokell/serokell-util` repo. I would like it to be updated. And it can be applied lazily to touched files.